Library/Formula/cyassl.rb

```diff
@@ -14,39 +12,55 @@ class Cyassl < Formula
     sha1 "c5de09829f89696a73a8f3818bbf413eae99e5ac" => :mountain_lion
   end
 
+  option "without-check", "Skip compile-time tests. Saves 20 seconds."
+  option "with-aesni", "Build with support for AES-NI, which most >i5 CPUs support"
```

Comment:
Might be nice to enable this by default on those machines? Poke around in the `Hardware` class to see how.